npx amplify-app@latest fails with 'Error: Cannot find module 'graphql'

**Describe the bug**
When following the Amplify DataStore instructions, the `npx amplify-app@latest` step fails with:
'Error: Cannot find module 'graphql'

**Amplify CLI Version**
Same issue on 4.13.4 and 4.13.2

**To Reproduce**
1) Create a new react project:
```
npx create-react-app my-app
cd my-app
```
2) Run the amplify-app command:
```
npx amplify-app@latest
```
3) See the error:
```
✅ Found Amplify CLI version 4.13.4
🎸 Creating base Amplify project
Error: Cannot find module 'graphql'
Require stack:
- /usr/local/lib/node_modules/amplify-app/node_modules/graphql-import/dist/index.js
- /usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/utils.js
- /usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/getGraphQLConfig.js
- /usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/index.js
- /usr/local/lib/node_modules/amplify-app/node_modules/amplify-frontend-javascript/lib/frontend-config-creator.js
- /usr/local/lib/node_modules/amplify-app/node_modules/amplify-frontend-javascript/index.js
- /usr/local/lib/node_modules/amplify-app/src/index.js
- /usr/local/lib/node_modules/amplify-app/bin/amplify-app
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:794:15)
    at Function.Module._load (internal/modules/cjs/loader.js:687:27)
    at Module.require (internal/modules/cjs/loader.js:849:19)
    at require (internal/modules/cjs/helpers.js:74:18)
    at Object.<anonymous> (/usr/local/lib/node_modules/amplify-app/node_modules/graphql-import/dist/index.js:15:17)
    at Module._compile (internal/modules/cjs/loader.js:956:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:973:10)
    at Module.load (internal/modules/cjs/loader.js:812:32)
    at Function.Module._load (internal/modules/cjs/loader.js:724:14)
    at Module.require (internal/modules/cjs/loader.js:849:19) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/usr/local/lib/node_modules/amplify-app/node_modules/graphql-import/dist/index.js',
    '/usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/utils.js',
    '/usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/getGraphQLConfig.js',
    '/usr/local/lib/node_modules/amplify-app/node_modules/graphql-config/lib/index.js',
    '/usr/local/lib/node_modules/amplify-app/node_modules/amplify-frontend-javascript/lib/frontend-config-creator.js',
    '/usr/local/lib/node_modules/amplify-app/node_modules/amplify-frontend-javascript/index.js',
    '/usr/local/lib/node_modules/amplify-app/src/index.js',
    '/usr/local/lib/node_modules/amplify-app/bin/amplify-app'
  ]
}
```

**Expected behavior**
Command should run without errors

**Desktop (please complete the following information):**
 - OS: macOS 10.14.6
 - Node Version: v12.12.0

I can reproduce it 

```
npx: installed 111 in 6.298s
✅ Found Amplify CLI version 4.13.4
🎸 Creating base Amplify project
{ Error: Cannot find module 'graphql'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:636:15)
    at Function.Module._load (internal/modules/cjs/loader.js:562:25)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (/Users/ammkara/.npm/_npx/46624/lib/node_modules/amplify-app/node_modules/graphql-import/dist/index.js:15:17)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3) code: 'MODULE_NOT_FOUND' }
```

--- a/packages/amplify-app/package.json
+++ b/packages/amplify-app/package.json
@@ -33,6 +33,7 @@
     "amplify-frontend-javascript": "2.13.2",
     "chalk": "^3.0.0",
     "fs-extra": "^8.1.0",
+    "graphql": "^14.5.8",
     "ini": "^1.3.5",
     "inquirer": "^7.0.3",
     "node-emoji": "^1.10.0",

